### PR TITLE
Ria 3307 ftpa decided notification

### PR DESCRIPTION
**Pull request by jkumar-moj**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RIA-3307

### Change description ###
ftpa decided notification trigger

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/LeadershipJudgeFtpaDecisionConfirmationTest.java [hunk 1]
```diff
@@ -5,9 +5,7 @@
 import static org.hamcrest.Matchers.containsString;
 import static org.junit.Assert.*;
 import static org.mockito.Mockito.*;
-import static uk.gov.hmcts.reform.iacaseapi.domain.entities.AsylumCaseFieldDefinition.FTPA_APPELLANT_DECISION_OUTCOME_TYPE;
-import static uk.gov.hmcts.reform.iacaseapi.domain.entities.AsylumCaseFieldDefinition.FTPA_APPLICANT_TYPE;
-import static uk.gov.hmcts.reform.iacaseapi.domain.entities.AsylumCaseFieldDefinition.UPLOAD_HOME_OFFICE_BUNDLE_ACTION_AVAILABLE;
+import static uk.gov.hmcts.reform.iacaseapi.domain.entities.AsylumCaseFieldDefinition.*;
```
Comment (Lee365) [inline src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/LeadershipJudgeFtpaDecisionConfirmationTest.java:8]: AsylumCaseFieldDefinition.* 
Explicit definition is suggested.

File: src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/ResidentJudgeFtpaDecisionConfirmationTest.java [hunk 2]
```diff
@@ -5,9 +5,7 @@
 import static org.hamcrest.Matchers.containsString;
 import static org.junit.Assert.*;
 import static org.mockito.Mockito.*;
-import static uk.gov.hmcts.reform.iacaseapi.domain.entities.AsylumCaseFieldDefinition.FTPA_APPELLANT_RJ_DECISION_OUTCOME_TYPE;
-import static uk.gov.hmcts.reform.iacaseapi.domain.entities.AsylumCaseFieldDefinition.FTPA_APPLICANT_TYPE;
-import static uk.gov.hmcts.reform.iacaseapi.domain.entities.AsylumCaseFieldDefinition.UPLOAD_HOME_OFFICE_BUNDLE_ACTION_AVAILABLE;
+import static uk.gov.hmcts.reform.iacaseapi.domain.entities.AsylumCaseFieldDefinition.*;
```
Comment (Lee365) [inline src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/ResidentJudgeFtpaDecisionConfirmationTest.java:8]: AsylumCaseFieldDefinition.*
Explicit definition is suggested.

File: src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/LeadershipJudgeFtpaDecisionConfirmationTest.java (hunk 1 above)
Comment (jkumar-moj) [reply src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/LeadershipJudgeFtpaDecisionConfirmationTest.java:8]: case-api does allow this formatting and is used everywhere. May be it is a wider thing to be consistent across.

File: src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/ResidentJudgeFtpaDecisionConfirmationTest.java (hunk 2 above)
Comment (jkumar-moj) [reply src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/ResidentJudgeFtpaDecisionConfirmationTest.java:8]: case-api does allow this formatting. May be it is a wider thing to be consistent across.

**Review by Lee365 — APPROVED** [review]
LGTM

**Review by Srihari-Nakkala — APPROVED** [review]
LGTM

**Review by makowalski — APPROVED** [review]
LGTM

**Pull request merged**